integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/AddSmartMeter.feature

```diff
@@ -21,12 +21,15 @@ Feature: SmartMetering Installation - Add smart meter
       | Encryption_key        | SECURITY_KEY_E    |
       | ManufacturerCode      | Test              |
       | ModelCode             | Test              |
+      | Timezone              | Europe/Amsterdam  |
```

Comment:
Also add a test without timezone

Reply:
Added test for E-meter without Timezone. Added check for empty timezone in AddMbusDevice.feature